tests/test_Subfunction_Module.py

```diff
@@ -0,0 +1,320 @@
+import numpy as np  # pragma: no cover
+import pytest  # pragma: no cover
+from scipy.signal.windows import triang  # pragma: no cover
+
+from Subfunction_Module import (
+    buscamin,
+    modmax,
+    picant,
+    picpost,
+    searchoff,
+    searchon,
+    zerocros,
+)  # pragma: no cover
+
+
+@pytest.mark.buscamin_testmarker
+def test_buscamin():
+    """
+    Test for the "buscamin" function;
+    Located in the Subfunction Module of py_pecg.
+    """
+
+    # Valid Cases
+    Test_Signal_1 = np.zeros(10, dtype=int)
+    assert buscamin(Test_Signal_1) == 0
```

Comment:
In my opinion, would be clearer as a 1 liner:
```suggestion
    assert buscamin(np.zeros(10, dtype=int)) == 0
```